src/AppBundle/Search/SearchParametersHelper.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace AppBundle\Search;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Provides a way to handle the search parameters.
+ */
+class SearchParametersHelper
```

Comment:
It's up to you, absolutely no problem. But, `util` is more common to group functions, and `filter` also... I would like to find a name that allow to easily understand that this object *store* the status of the search parameters in addition to help for the parameters handling...